v2/cli/init_command.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/brigadecore/brigade/sdk/v2/core"
+	"github.com/brigadecore/brigade/v2/internal/file"
+	"github.com/urfave/cli/v2"
+)
+
+type initFields struct {
+	ProjectID   string
+	Language    string
+	GitCloneURL string
+	Script      string
+}
+
+var initCommand = &cli.Command{
+	Name:  "init",
+	Usage: "Initialize a new Brigade project",
+	Flags: []cli.Flag{
+		&cli.StringFlag{
+			Name:    flagID,
+			Aliases: []string{"i"},
+			Usage: "Name of the Brigade project to initialize " +
+				"(required)",
+			Required: true,
+		},
+		&cli.StringFlag{
+			Name:    flagLanguage,
+			Aliases: []string{"l"},
+			Usage: "Specify the scripting language to use for handling events" +
+				" -- JavaScript (js) or TypeScript (ts)",
+			Value: "ts",
+		},
+		&cli.StringFlag{
+			Name:    flagGit,
+			Aliases: []string{"g"},
+			Usage: "Optionally specify a git repository where the" +
+				" event handling script can be found",
+		},
+	},
+	Action: initialize,
+}
+
+// Handle init CLI flags
```

Comment:
I think I mentioned that for exported functions, the comment above should start with the name of the function. While that's not enforced for unexported functions, it's still a good thing to adhere to-- that way comments don't have to be significantly re-written if something that was once unexported becomes exported in the future. In this case, it would also be fine to just drop the comment.